web/docs/use/transform/README.md

```diff
@@ -205,22 +207,22 @@ the `compaction` subcommand. Use the `list` subcommand to show all configured
 compaction rules:
 
 ```bash
-vast compaction list
+tenzirctl compaction list
 ```
 
 You can then trigger a compaction manually with the `run` sub-command:
 
 ```bash
-vast compaction run <rule>
+tenzirctl compaction run <rule>
 ```
 
 :::note
-The `compaction` plugin needs to be loaded both by the client and the server process to
-use the `vast compaction` subcommand.
+The `compaction` plugin needs to be loaded both by the client and the server
+process to use the `tenzirctl compaction` subcommand.
 :::
 
-For an overview of the current status of the compaction plugin, you can use the `vast status` subcommand:
+For an overview of the current status of the compaction plugin, you can use the `tenzirctl status` subcommand:
 
 ```bash
-vast status | jq .compaction
+tenzirctl status | jq .compaction
```

Comment:
```suggestion
tenzirctl status compaction
```

We've supported this for quite a while now in the `status` command to just fetch a subset.